jumanji/environments/combinatorial/binpack/env_viewer.py

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from enum import Enum
```

Comment:
[Import modules please](https://google.github.io/styleguide/pyguide.html#22-imports).

Reply:
Sure, I always forget this one sorry